genie_registry/maf.py

```diff
@@ -275,6 +275,18 @@ def _validate(self, mutationDF):
             )
             total_error.write(error)
 
+        # only check end position as start position is required col
+        if process_functions.checkColExist(
```

Comment:
I didn't noticed this, but perhaps we should add the `START_POSITION` and `END_POSITION` in the numerical value check above to make sure these are numerical values.  This way you don't need to update the function to convert these to integers. 

There will be a validation error if a non-numerical value gets passed into START or END position.

Reply:
Ah I was wondering about that. I saw that happening for another file format namely `bed` so I thought `maf` didn't need it. I can add that.